packages/sdk-v2/src/sdk/goodid/sdk.ts

```diff
@@ -0,0 +1,65 @@
+import { Envs } from "../constants";
+import { CertificateItem } from "./types";
+
+const getBearerToken = async (baseEnv: string, fvsig: string, account: string, type: "location" | "identity") => {
+  const devEnv = baseEnv === "fuse" ? "development" : baseEnv;
+  const { backend } = Envs[devEnv];
+
+  const endpoint = `${backend}/goodid/certificate/${type}`;
+  const authEndpoint = `${backend}/auth/fv2`;
+  const { token } = await fetch(authEndpoint, {
```

Comment:
we already have it, at faucet or delete fv call, please reuse

Reply:
as I remember we have separate function to auth. just re-call it from here

Reply:
```typescript
const fvAuth = async (signer: Signer, env: EnvKey): Promise<string> => {
  const { backend } = Envs[env];
  const authEndpoint = `${backend}/auth/fv2`;
  
  const account = await signer.getAddress();
  const fvsig = await signer.signMessage(FV_IDENTIFIER_MSG2.replace("<account>", account));
  const { token } = await fetch(authEndpoint, g$Request({ fvsig, account })).then(g$Response)
  
  return token
}
```

1. reuse it at both deleteFVToken and requestXXXCertificate
2. instead of getBearerToken you could define a simple helper such as `const getDevEnv = (baseEnv: EnvKey) =>  baseEnv === "fuse" ? "development" : baseEnv;`. maybe this code repeats in other places so you could reuse such util also there
3. then at request cert methods you do
```javascript
const env = getDevEnv(baseEnv)
const { backend } = Envs[env]; 
const token = await fvAuth(signer, env)

return fetch(
  `${backend}/goodid/certificate/location`, // or identity
  g$AuthRequest({ // data corresponding to cert type })
).then(g$Response)
```

Reply:
implemented here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/136/commits/f79359a2cc60f3409328fd0fb1370bd7078ec57e